src/test/java/de/iav/mucjava231burgershop/controller/BurgerShopControllerTest.java

```diff
@@ -0,0 +1,63 @@
+package de.iav.mucjava231burgershop.controller;
+
+import de.iav.mucjava231burgershop.model.Menu;
+import de.iav.mucjava231burgershop.repository.MenuRepository;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class BurgerShopControllerTest {
```

Comment:
```suggestion
class BurgerShopControllerIntegrationTest {
```

Das man direkt sieht das es ein IntegrationTest ist